migration/migrator/data/course_tables.sql

```diff
@@ -1533,7 +1533,9 @@ CREATE TABLE public.polls (
     image_path text,
     question_type character varying(35) DEFAULT 'single-response-multiple-correct'::character varying,
     release_histogram character varying(10) DEFAULT 'never'::character varying,
-    release_answer character varying(10) DEFAULT 'never'::character varying
+    release_answer character varying(10) DEFAULT 'never'::character varying,
+    duration character varying(255) DEFAULT 'P0D'::character varying,
```

Comment:
This should be converted to a proper interval type before this PR can be merged.  If that turns out to be challenging, you could store it as integer seconds if necessary.